### PR TITLE
make this as dummy operator in cs 3.8

### PR DESCRIPTION
**Pull request by Tirumalavasa**
Code prior to this dummy operator is present in `master-orig-cs39` branch

### CODE REVIEW

**Comment by pgodowski** [comment]
please have a review with Tamilu too. Please before also create branch/tag/release before this change gets merged.

File: pkg/controller/prometheusext/prometheusext_controller.go [hunk 1]
```diff
@@ -142,22 +139,6 @@ func (r *ReconcilePrometheusExt) Reconcile(request reconcile.Request) (reconcile
 		// Error reading the object - requeue the request.
 		return reconcile.Result{}, err
 	}
-	reconsiler := reconsiler.Reconsiler{
```
Comment (pgodowski) [inline pkg/controller/prometheusext/prometheusext_controller.go:145]: given that it's dummy operator now, shall we revisit the resource request/limits for the operator, to lower it down?

**Review by pgodowski — CHANGES_REQUESTED** [review]

**Review by pgodowski — APPROVED** [review]

File: pkg/controller/prometheusext/prometheusext_controller.go (hunk 1 above)
Comment (Tirumalavasa) [reply pkg/controller/prometheusext/prometheusext_controller.go:145]: Sure, i will adjust these resources in the csv in the upcoming PR

**Pull request merged**